h/views/organizations.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from pyramid.view import view_config
+
+
+@view_config(route_name='organization_logo', request_method='GET', renderer='svg')
+def organization_logo(logo, request):
+    return logo
```

Comment:
.... Aaand finally the view itself. By the time we get here there's nothing left to do!